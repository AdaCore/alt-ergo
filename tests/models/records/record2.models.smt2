(set-logic ALL)
(set-option :produce-models true)
(declare-datatype Pair
((pair (first Int) (second Int))))
(declare-fun f (Int) Pair)
(assert (= (first (f 0)) 5))
(check-sat)
(get-model)
