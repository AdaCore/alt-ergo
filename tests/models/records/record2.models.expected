
unknown
(
  (define-fun f ((_arg_0 Int)) Pair (pair 5 0))
)
